hw/ip/aes/pre_sca/alma/cpp/testbench.h

```diff
@@ -0,0 +1,73 @@
+// Copyright IAIK.
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "verilated.h"
+#include "verilated_vcd_c.h"
+
+template <class Module>
+struct Testbench {
+  unsigned long m_tickcount;
+  Module *m_core;
+  VerilatedVcdC *m_trace;
+
+  Testbench(void) {
+    Verilated::traceEverOn(true);
+    m_core = new Module;
```

Comment:
Any particular reason that `m_core` is a pointer here? I would have thought you could have just declared it as `Module m_core`, which would let you replace a whole load of `->` with `.` and get rid of the destructor entirely.

Reply:
Agreed and done.